airbyte-webapp/src/pages/ConnectionPage/pages/ConnectionItemPage/ConnectionSettingsTab/ConnectionSettingsTab.tsx

```diff
@@ -23,6 +24,7 @@ export const ConnectionSettingsTab: React.FC = () => {
 
   return (
     <div className={styles.container}>
+      <SchemaUpdateNotifications />
```

Comment:
this needs to only be rendered if the feature is turned on

Reply:
Fixed. Thanks!